packages/cli/src/utils/checkMissingAppInfo.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async function (app: Object) {
```

Comment:
Does this function have to be `async`? I would expect every line in this function to run synchronously.